code/modules/mob/living/carbon/human/species/monkey.dm

```diff
@@ -33,6 +33,14 @@
 
 	flags = IS_RESTRICTED
 
+
+/mob/living/carbon/human/monkey/prepare_data_huds()
+	//Prepare our med HUD...
+	..()
+	//...and display it.
+	for(var/datum/atom_hud/data/human/medical/hud in huds)
+		hud.add_to_hud(src)
```

Comment:
just a comment for this; since monkies are human on our server vs TG (which are still just carbons), is this still necessary? Again, just checking.